### ISSUE TITLE
Incorrect TypeScript Definition for sentences() 

### ISSUE THREAD

**Issue by MarkHerhold** [spencermountain/compromise#756]
I believe there is a TypeScript definition issue with `sentences()`:

Error:
```
Property 'subjects' does not exist on type 'ExtendedDocument<{ sentences(n?: number | undefined): Sentences; }, World, Phrase>'.
```

TS Code:
```ts
import nlp from 'compromise';
import compromiseSentences from 'compromise-sentences';
const nlpEx = nlp.extend(compromiseSentences);

nlpEx('Hi bob').sentences().subjects();
```

Working on observablehq:
![image](https://user-images.githubusercontent.com/4934732/83169899-b3a14d80-a0e1-11ea-853a-b7ba1d6224f9.png)

**Comment by spencermountain** [comment]
ah, thank you. good find.
will add to next release.

**Comment by spencermountain** [comment]
yeah, it looks like typescript doesn't see the new methods from the plugin, can reproduce.
@Drache93 any advice?
cheers

**Comment by Drache93** [comment]
Ah yeh. @spencermountain Odd one, for some reason `sentence()` shows an overload, so it returns an ExtendedDocument if no parameters, otherwise returns Sentence correctly...I'll poke around!

**Comment by Drache93** [comment]
Oh right :D `sentences(): ExtendedDocument<Ext, W, Ph>` is a built in function (looks like it should actually take a parameter also...).

**Comment by spencermountain** [comment]
you're right - i added the missing param and I think it's properly overloaded now.
Thanks!
fixed in `13.3.2`
